src/services/geometry/richgeo/IrtGeoDRICH.cc

```diff
@@ -75,58 +73,58 @@ void richgeo::IrtGeoDRICH::DD4hep_to_IRT() {
         (G4LogicalVolume*)(0x2), // G4LogicalVolume (inaccessible? use an integer instead)
         nullptr,                 // G4RadiatorMaterial
         filterFlatSurface,       // surface
-        filterThickness / dd4hep::mm // surface thickness
+        filterThickness          // surface thickness
         );
     aerogelFlatRadiator->SetAlternativeMaterialName(aerogelMaterial.c_str());
     filterFlatRadiator->SetAlternativeMaterialName(filterMaterial.c_str());
   }
-  m_log.PrintLog("aerogelZpos = {:f} cm", aerogelZpos);
-  m_log.PrintLog("filterZpos  = {:f} cm", filterZpos);
-  m_log.PrintLog("aerogel thickness = {:f} cm", aerogelThickness);
-  m_log.PrintLog("filter thickness  = {:f} cm", filterThickness);
+  m_log->debug("aerogelZpos = {:f} mm", aerogelZpos);
+  m_log->debug("filterZpos  = {:f} mm", filterZpos);
+  m_log->debug("aerogel thickness = {:f} mm", aerogelThickness);
+  m_log->debug("filter thickness  = {:f} mm", filterThickness);
 
   // sector loop
   for (int isec = 0; isec < nSectors; isec++) {
     std::string secName = "sec" + std::to_string(isec);
 
     // mirrors
-    auto mirrorRadius = m_det->constant<double>("DRICH_mirror_radius");
+    auto mirrorRadius = m_det->constant<double>("DRICH_mirror_radius") / dd4hep::mm;
     dd4hep::Position mirrorCenter(
-      m_det->constant<double>("DRICH_mirror_center_x_"+secName),
-      m_det->constant<double>("DRICH_mirror_center_y_"+secName),
-      m_det->constant<double>("DRICH_mirror_center_z_"+secName)
+      m_det->constant<double>("DRICH_mirror_center_x_"+secName) / dd4hep::mm,
+      m_det->constant<double>("DRICH_mirror_center_y_"+secName) / dd4hep::mm,
+      m_det->constant<double>("DRICH_mirror_center_z_"+secName) / dd4hep::mm
       );
-    auto mirrorSphericalSurface  = new SphericalSurface(
-        (1 / dd4hep::mm) * TVector3(mirrorCenter.x(), mirrorCenter.y(), mirrorCenter.z()), mirrorRadius / dd4hep::mm);
+    auto mirrorSphericalSurface  = new SphericalSurface(TVector3(mirrorCenter.x(), mirrorCenter.y(), mirrorCenter.z()), mirrorRadius);
     auto mirrorOpticalBoundary = new OpticalBoundary(
         m_irtDetector->GetContainerVolume(), // CherenkovRadiator radiator
         mirrorSphericalSurface,            // surface
         false                              // bool refractive
         );
     m_irtDetector->AddOpticalBoundary(isec, mirrorOpticalBoundary);
-    m_log.PrintLog("");
-    m_log.PrintLog("  SECTOR {:d} MIRROR:", isec);
-    m_log.PrintLog("    mirror x = {:f} cm", mirrorCenter.x());
-    m_log.PrintLog("    mirror y = {:f} cm", mirrorCenter.y());
-    m_log.PrintLog("    mirror z = {:f} cm", mirrorCenter.z());
-    m_log.PrintLog("    mirror R = {:f} cm", mirrorRadius);
+    m_log->debug("");
+    m_log->debug("  SECTOR {:d} MIRROR:", isec);
+    m_log->debug("    mirror x = {:f} mm", mirrorCenter.x());
```

Comment:
Canonical for unit quantities:
```suggestion
    m_log->debug("    mirror x = {:f} mm", mirrorCenter.x() / dd4hep::mm);
```
Even if dd4hep changes internally to cm, it would still print in mm here.

Reply:
Dividing by `dd4hep::mm` already happened a few lines above:
https://github.com/eic/EICrecon/blob/433a93afc02377b90e79e7f33fcb71a146c2c39e/src/services/geometry/richgeo/IrtGeoDRICH.cc#L92-L96